bundle/src/main/java/com/adobe/acs/commons/wcm/vanity/impl/ExtensionlessRequestWrapper.java

```diff
@@ -20,65 +20,12 @@
 package com.adobe.acs.commons.wcm.vanity.impl;
 
 import org.apache.sling.api.SlingHttpServletRequest;
-import org.apache.sling.api.request.RequestPathInfo;
-import org.apache.sling.api.resource.Resource;
-import org.apache.sling.api.resource.ValueMap;
 import org.apache.sling.api.wrappers.SlingHttpServletRequestWrapper;
 
-public class ExtensionlessRequestWrapper extends SlingHttpServletRequestWrapper {
 
-    private static final String SLING_STATUS = "sling:status";
+public class ExtensionlessRequestWrapper extends SlingHttpServletRequestWrapper {
```

Comment:
what does this empty wrapper do now? IMHO all methods just delegate.... Can't we remove then the whole wrapper?

Reply:
Agreed -- I'm thinking I should re-work that PR to function more like the work done in this one. Thoughts?

Reply:
can you rebase this PR to exclude the RequestPathInfo changes that are present in the other PR?

Reply:
Rebased/removed the other stuff.